op-guide/dashboard-pd-info.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 重要监控指标详解
+category: monitoring
+---
+
+# 重要监控指标详解
+
+使用 Ansible 部署 TiDB 集群时，一键部署监控系统 (Prometheus/Grafana)，监控架构请看 [TiDB 监控框架概述](monitor-overview.md)。
+
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node\_exporter、Overview 等。
+
+对于日常运维，我们通过观察 PD 面板上的 Metrics，可以了解 PD 当前的状态。
+
+以下为 PD Dashboard 监控说明：
+
+## 说明
+
+- Cluster
+    - PD Role： 当前 PD 的角色
```

Comment:
Please remove the extra space after the Chinese colon. And also remove similar ones involved in this PR.